instrumentation/logback/logback-1.0.0/library/src/main/java/io/opentelemetry/instrumentation/logback/v1_0_0/UnionMap.java

```diff
@@ -29,14 +29,14 @@
  * An immutable view over two maps, with keys resolving from the first map first, or otherwise the
  * second if not present in the first.
  */
-final class UnionMap<K, V> extends AbstractMap<K, V> {
+public final class UnionMap<K, V> extends AbstractMap<K, V> {
```

Comment:
Can you move this into a subpackage `internal`? We shouldn't add this to our public API

Reply:
Done, thanks!